api/src/templates/privacy-policy/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="theme-color" content="#323232">
+    <link rel="shortcut icon" href="/favicon.ico">
+    <link rel="manifest" href="/manifest.json">
+    <style>
+      @font-face {
+        font-family: 'Noto';
+        src: url(/fonts/NotoSans-Regular.ttf) format("truetype");
+        font-weight: normal;
+      }
+      body {
+        font-family: Noto, sans-serif;
+      }
+      div {
+        max-width: 800px;
+        margin: 40px auto;
+      }
+      a {
+        padding: 20px;
+        margin: -20px;
+      }
+    </style>
+  </head>
+  <body>
+    <div>
+      <h1>{{ translations.title }}</h1>
+    </div>
+    <div>
+      {{ policy }}
+    </div>
+    <div>
+      <% if(showBackButton) { %>
```

Comment:
Is this best practice, to put the back or login button at the bottom only? Thinking about very long privacy policy texts on mobile.

Reply:
Copied the section to the top of the doc too.